go/vt/schema/online_ddl_test.go

```diff
@@ -27,24 +29,38 @@ func TestCreateUUID(t *testing.T) {
 	assert.NoError(t, err)
 }
 
-func TestValidateDDLStrategy(t *testing.T) {
-	{
-		strategy, err := ValidateDDLStrategy("gh-ost")
-		assert.NoError(t, err)
-		assert.Equal(t, DDLStrategyGhost, strategy)
+func TestParseDDLStrategy(t *testing.T) {
+	tt := []struct {
+		strategyVariable string
+		strategy         sqlparser.DDLStrategy
+		options          string
+		err              error
+	}{
```

Comment:
add an invalid parsing test.

Reply:
There is one, at the end of the function